test/vcs_cloner_test.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'git'
+
+require_relative '../lib/smart_proxy_ansible/vcs_cloner'
+
+class VcsClonerTest < Minitest::Test
+  describe '#install' do
+    test 'clones a role' do
+      repo_info = {
+        'vcs_url' => 'https://www.github.com/theforeman/theforeman.git',
```

Comment:
How is this success if this repo doesn't exist? (404 not found)

Reply:
I mocked the git method in this test, so the url didn't really matter. 
Though, it obviously makes more sense to put in a valid git url :)